### PR TITLE
Make compatible with auto_now(_add)

### LINKED ISSUE #33: DateTimeField with auto_add_now always has now value
I have a `DateTimeField` with `auto_add_now=True`. 
When I run 
```
seeder.add_entity(Item, 15, {'date': lambda x: seeder.faker.date_time_between(start_date='-1y'),})
seeder.execute()
```
(or without defining the `date` generator) I only get Items with the date equal to the time I run the code.

I believe this is incorrect behaviour, please correct me if I'm wrong.

### PR DESCRIPTION
**Pull request by jgadelange**
Fixes #33 

Also some minor style stuff

### CODE REVIEW

**Comment by Brobin** [comment]
Thanks Jeffrey! I'm on vacation right now but will look at this more
carefully when I get back.

**Comment by Brobin** [comment]
Looks great! Could you perhaps add a test with an `auto_now` or `auto_now_add` field to validate that it works?

File: django_seed/tests.py [hunk 1]
```diff
@@ -146,6 +148,30 @@ def test_no_entities_added(self):
...
+        date =  datetime(1957, 3, 6, 13, 13)
+        faker = fake
+        seeder = Seeder(faker)
+        seeder.add_entity(Game, 10, {
+            'created_at': lambda x: date
+        })
+        seeder.execute()
+
+        games = Game.objects.all()
+        self.assertTrue(any(game.created_at == date for game in games))
```
Comment (Brobin) [inline django_seed/tests.py]: Shouldn't `any` by `all`? Otherwise, looks good.

Comment (jgadelange) [inline django_seed/tests.py]: You are right. DOH!
Fixed it

**Comment by Brobin** [comment]
Merging! I'll get this pushed up in the next release

**Pull request merged**